test/hotspot/jtreg/serviceability/HeapDump/UnmountedVThreadNativeMethodAtTop.java

```diff
@@ -27,7 +27,9 @@
  * @requires vm.continuations
  * @modules jdk.management
  * @library /test/lib
- * @run junit/othervm --enable-native-access=ALL-UNNAMED UnmountedVThreadNativeMethodAtTop
+ * @build jdk.test.whitebox.WhiteBox
+ * @run driver jdk.test.lib.helpers.ClassFileInstaller jdk.test.whitebox.WhiteBox
+ * @run junit/othervm -Xbootclasspath/a:. -XX:+UnlockDiagnosticVMOptions -XX:+WhiteBoxAPI --enable-native-access=ALL-UNNAMED UnmountedVThreadNativeMethodAtTop
```

Comment:
Maybe use '@run junit/othervm/native' more better.

Reply:
I guess that's better. Thanks for the hint. I wasn't aware of the `/native` option.